test/IRGen/protocol_conformance_records.swift

```diff
@@ -136,7 +157,7 @@ extension Int : OtherResilientProtocol { }
 // -- witness table pattern
 // CHECK-SAME:           @"$s28protocol_conformance_records9DependentVyxGAA9AssociateAAWp"
 // -- flags
-// CHECK-SAME:           i32 0
+// CHECK-SAME:           i32 131072,
```

Comment:
Note that this is not a change in test output -- the test was matching the wrong "i32 0"